### PR TITLE
Vet clinic database: query multiple tables

### PR DESCRIPTION
**Pull request by najibullahjafari**
here:

- I create a table named owners with the following columns:

1. id: integer (set it as autoincremented PRIMARY KEY)
2. full_name: string
3. age: integer

- I created a table named species with the following columns:

1. id: integer (set it as autoincremented PRIMARY KEY)
2. name: string
3. Modify animals table
- I removed column species
- I added column species_id which is a foreign key referencing species table
- I added column owner_id which is a foreign key referencing the owners table

- I inserted the following data into the owners table:

- [ ] Sam Smith 34 years old.
- [ ] Jennifer Orwell 19 years old.
- [ ] Bob 45 years old.
- [ ] Melody Pond 77 years old.
- [ ] Dean Winchester 14 years old.
- [ ] Jodie Whittaker 38 years old.

- I inserted the following data into the species table:

- [ ] Pokemon
- [ ] Digimon

- I  modified my inserted animals so it includes the species_id value:

- [ ] If the name ends in "mon" it will be Digimon
- [ ] All other animals are Pokemon

- I modified my inserted animals to include owner information (owner_id):

- [ ] Sam Smith owns Agumon.
- [ ] Jennifer Orwell owns Gabumon and Pikachu.
- [ ] Bob owns Devimon and Plantmon.
- [ ] Melody Pond owns Charmander, Squirtle, and Blossom.
- [ ] Dean Winchester owns Angemon and Boarmon.

- I answered these questions:

1. What animals belong to Melody Pond?
2. List of all animals that are pokemon (their type is Pokemon).
3. List all owners and their animals, remember to include those that don't own any animal.
4. How many animals are there per species?
5. List all Digimon owned by Jennifer Orwell.
6. List all animals owned by Dean Winchester that haven't tried to escape.
7. Who owns the most animals?
Here you can see the result of queries:
Screenshot 2023-08-31 163808
Screenshot 2023-08-31 163949
Screenshot 2023-08-31 164445
Screenshot 2023-08-31 165548
Screenshot 2023-08-31 165911
Screenshot 2023-08-31 170613
Screenshot 2023-08-31 171336

### CODE REVIEW

**Review by Gambit142 — COMMENTED** [review]
Hi @najibullahjafari 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- Kindly add the Future Features section in the README file. The Future Features should contain 1 - 3 features that can be implemented in this project at a later time. 👍 😎 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: data.sql [hunk 1]
```diff
@@ -1,34 +1,87 @@
...
+
+
+
+-- inserting DATABASE
+INSERT INTO owners (full_name, age) VALUES ('Sam Smith', 34);
+INSERT INTO owners(full_name, age) VALUES ('Jennifer Orwell', 19);
+INSERT INTO owners(full_name, age) VALUES ('Bob', 45);
+INSERT INTO owners(full_name, age) VALUES ('Melody pond', 77);
+
+INSERT INTO owners(full_name, age) VALUES ('Deian watchster', 14);
```
Comment (BregornOriginal) [inline data.sql]: - [ ] Here's a typo in this `full_name`. Kindly modify it to the correct value `Dean Winchester`

File: data.sql [hunk 2]
```diff
@@ -1,34 +1,87 @@
...
+
+INSERT INTO owners(full_name, age) VALUES ('Jodie Wittakar', 38);
```
Comment (BregornOriginal) [inline data.sql]: - [ ] Here's a typo in this `full_name`. Kindly modify it to the correct value `Jodie Whittaker`

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
Hi @najibullahjafari  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

giphy (3)

## Highlights

✅ Linters are passing
✅ Good Pull Request description
✅ README looks professional

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by iambenkis — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- Update according to requirements :heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @iambenkis in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**